### ISSUE TITLE
Credit card entry form hangs after configuration changes

### ISSUE THREAD

**Issue by idomo1** [braintree/braintree-android-drop-in#348]
### General information

* SDK/Library version: 6.1.0
* Environment: Sandbox
* Android Version and Device: (non-exhaustive list) Samsung Note 5 API 24, API 29 (emulator)

### Issue description

If any configuration changes happen while the Credit or Debit card form is open, after submitting it will just go into an infinite load and never redirect back to the app.

- Launch the drop in UI payment picker
- Select "Credit or Debit Card"
- Perform a configuration change (change orientation, toggle night mode, change font size, etc)
- Fill in the form and try to submit
- A loading indicator appears, but nothing happens after that

The Paypal and Google Pay methods don't seem to have this issue.

A workaround is to lock the orientation/ignore config changes using manifest merging

```
<activity android:name="com.braintreepayments.api.DropInActivity"
            android:screenOrientation="locked"
            android:configChanges="uiMode|fontScale|..."
            tools:replace="configChanges,screenOrientation" />
```

But this isn't a good solution, since then config changes have no affect (obviously).

**Comment by sshropshire** [comment]
Hi @idomo1 thanks for using the Braintree SDK for Android. We've confirmed the issue and implemented a fix. Once we get it approved and merged it will be released shortly after.

**Comment by sshropshire** [comment]
Released in version `6.2.0`.
